articles/tokens/access-token.md

```diff
@@ -0,0 +1,142 @@
+- The set of claims contains verifiable security statements such as the identity of the user and the permissions they are allowed.
+- The signature is used to validate that the token is trustworthy and has not been tampered with.
+
+It should be noted that other token formats are also available. For example, Auth0 Management API v1 (which has been deprecated) uses an opaque token format in which claims are referenced in a separate database, rather than directly in the token. Simple Web Token is an example of another token format, although it has not seen widespread adoption. Support for future tokens formats will be implemented if and when industry consensus develops around them.
+
+## How to get an access token
+
+Access tokens are issued via [Auth0's OAuth 2.0 endpoints](/api/authentication): `/authorize` and `/oauth/token`. You can use any OAuth 2.0-compatible library to obtain access tokens. If you do not already have a preferred OAuth 2.0 library, Auth0 provides libraries for many language and frameworks that work seamlessly with our endpoints.
+
+* Calls to the Lock widget will return an `access_token` as shown in the [Lock documentation](/libraries/lock).
+* [Examples using auth0.js](https://github.com/auth0/auth0.js).
+* Check the [List of tutorials](/tutorials) to see how to make calls to libraries for other languages/SDKs.
+
+## How to use an access token
+
+Access tokens are typically obtained in order to access user-owned resources. For example, a Calendar application on a mobile phone needs access to a Calendar API in the cloud in order to read the user's scheduled events and create new events.
+
+Such access is requested by the application and granted by the user. For example, when using Lock, access to read and write the user's calendar can be requested by calling `show()`:
+
+```js
+lock.show({
+  responseType: 'token',
+  authParams: {
+    scope: 'read write'
+  }
+});
+```
+
+In this case the user will be prompted to permit read and write access. If allowed, an access token will be issued to the client, which the client can then use when making requests to the Calendar API. If consent has already been granted by the user, no consent dialog will be displayed and the access token will be issued without additional prompts.
+
+Also, the consent dialog might be displayed again if the access level changes. For example, if the user has granted read access but the functionality changes so write access is required as well, the user will have to use the consent dialog to grant the additional access.
+
+In some cases, consent can also be pre-configured administratively. This typically occurs when the user is an employee, and there are a set of company-specific applications that are always allowed access to employee data.
+
+### Server-to-server interactions
+
+Access tokens can also be issued directly to applications, permitting access to APIs that don't provide user-specific data. Such scenarios involve server-to-server interactions.
```

Comment:
I would rephrase to something like this - "Access tokens can also be issued directly to applications without an actual user needing to authenticate". It's perfectly feasible to have a single API that is called in the same way by both users and headless applications.